api/compose/api.go

```diff
@@ -169,6 +171,12 @@ type RunOptions struct {
 	Index int
 }
 
+// EventsOptions group options of the Events API
+type EventsOptions struct {
+	Services []string
+	Consumer func(timestamp time.Time, service, container, status string, attributes map[string]string) error
```

Comment:
We should add a struct for all these params to avoid changing signature if we change this (long) list of params